podcastfy/client.py

```diff
@@ -97,6 +103,9 @@ def main(
 	transcript: typer.FileText = typer.Option(
 		None, "--transcript", "-t", help="Path to a transcript file"
 	),
+	raw_file: typer.FileText = typer.Option(
+		None, "--raw-file", "-r", help="File containing raw text"
```

Comment:
this is a bit confusing: the user is passing raw text but the flag is raw file. It should be consistent. Consider renaming the flag to simply --text